lib/entities/quote/DutchQuote.ts

```diff
@@ -265,13 +312,19 @@ export class DutchQuote implements IQuote {
       endAmountIn: this.amountInEnd.toString(),
       endAmountOut: this.amountOutEnd.toString(),
       amountInGasAdjusted: this.amountInStart.toString(),
+      amountInGasAndPortionAdjusted: this.request.info.type === TradeType.EXACT_OUTPUT ? this.amountInGasAndPortionAdjusted.toString() : undefined,
       amountOutGasAdjusted: this.amountOutStart.toString(),
+      amountOutGasAndPortionAdjusted: this.request.info.type === TradeType.EXACT_INPUT ? this.amountOutGasAndPortionAdjusted.toString() : undefined,
       swapper: this.swapper,
       filler: this.filler,
       routing: RoutingType[this.routingType],
       slippage: parseFloat(this.request.info.slippageTolerance),
       createdAt: this.createdAt,
       createdAtMs: this.createdAtMs,
+      portionBips: this.portionBips,
+      portionRecipient: this.portionRecipient,
+      portionAmountOutStart: this.portionAmountOutStart.toString(),
+      portionAmountOutEnd: this.portionAmountOutEnd.toString(),
```

Comment:
maybe rename to `portionAmountStart` and `portionAmountEnd`? assuming we are going to reuse them for both exact_in and out trades